src/ast/astExtractor/expression-extractor.ts

```diff
@@ -359,7 +359,7 @@ export class ExpressionExtractor extends BaseJavaCstVisitorWithDefaults {
         kind: "Literal",
         literalType: {
           kind: "NullLiteral",
-          value: ctx.Null[0].image,
+          value: ctx.Null,
```

Comment:
Actl can remove this whole block since there is already a check for `ctx.Null` previously. I think there was duplication from the previous PR.

Reply:
Also, if you choose to remove this block, remember to extract the location of `ctx.Null` in the previous block as not doing so will fail one of the existing test case.

Reply:
Got it, removed.